README.md

```diff
@@ -49,15 +49,42 @@ The json must have the following fields:
 This file is required for the coverage integration so it needs to be added
 to the repository as well.
 
-1. Create a new pipeline definition in Buildkite. For this step ask one of the
-rust-vmm Buildkite [admins](CODEOWNERS) to create one for you. Add a pipeline
-step that is uploading the rust-vmm-ci pipeline:
+3. Create a new pipeline definition in Buildkite. For this step ask one of the
```

Comment:
Can you add a link to the process from the community repo? Here is the link: https://github.com/rust-vmm/community/blob/master/docs/maintainers/setup_new_repo.md#set-up-ci

Reply:
Added the link.